src/ArchiveManagement/NexusMods.FileExtractor/build/NexusMods.FileExtractor.targets

```diff
@@ -13,12 +13,8 @@
     <_NativeRuntime Condition=" '$(_NativeRuntime)' == '' And '$(_IsWindows)' == 'true' And ('$(Prefer32Bit)' == 'true' Or '$(PlatformTarget)' == 'x86')">win-x86</_NativeRuntime>
     <_NativeRuntime Condition=" '$(_NativeRuntime)' == '' And '$(_IsWindows)' == 'true' And ('$(Prefer32Bit)' == 'false' Or '$(PlatformTarget)' == 'x64')">win-x64</_NativeRuntime>
 
-      <_UseSystemExtractor Condition="'$(NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR)' == '1'">true</_UseSystemExtractor>
+    <_UseSystemExtractor Condition="$([System.String]::new('$(DefineConstants)').Contains('USE_SYSTEM_EXTRACTOR')) or $([System.String]::new('$(DefineConstants)').Contains('NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR'))">true</_UseSystemExtractor>
   </PropertyGroup>
-    
-    <PropertyGroup Condition="'$(_UseSystemExtractor)' == 'true'">
-        <DefineConstants>USE_SYSTEM_EXTRACTOR</DefineConstants>
-    </PropertyGroup>
 
```

Comment:
The property can be removed if it's not used.

Reply:
The property is used below to determine if to copy the 7zz binaries to output.